Tests/CommandsTests/TestCommandTests.swift

```diff
@@ -242,9 +232,7 @@ struct TestCommandTests {
                 #expect(result.stderr.contains("-enable-testing"))
             }
         } when: {
-            (buildSystem == .swiftbuild && .linux == ProcessInfo.hostOperatingSystem)
-            // || (buildSystem == .swiftbuild && .windows == ProcessInfo.hostOperatingSystem && CiEnvironment.runningInSelfHostedPipeline)
-            || (buildSystem == .swiftbuild && .windows == ProcessInfo.hostOperatingSystem )
+            buildSystem == .swiftbuild
```

Comment:
issue (possibly-blocking): this seems like a regression as this was previously passing on macOs, but is not failing.  Why is the plan to address this?